packages/data-store/src/configs.ts

```diff
@@ -0,0 +1,74 @@
+import {DataStoreConfigs, SDKConfigs} from './types/types';
+import {getDataSource} from './typeorm';
+import {SDKConfigsEntity} from './typeorm/entities/sdk-configs.entity';
+
+export async function getSDKConfigs(
+  options: DataStoreConfigs,
+): Promise<SDKConfigs> {
+  let result = await getDataSource().getRepository(SDKConfigsEntity).find();
+
+  if (result.length === 1) {
+    return result[0];
+  }
+
+  if (result.length > 1) {
+    throw new Error('More than one SDKConfigsEntity found');
```

Comment:
how would the user recover from this state? I'm assuming this could potentially make the app unusable?

Reply:
Maybe we can return only the first item, and log a warning on sentry, allowing us to track this state without braking the app.

Reply:
Created a follow up sub task https://dock-team.atlassian.net/browse/DCKA-1624

Reply:
We should probably try to add some safety during insert to ensure we aren't allowing a second one to be added in the first place.